public/app/features/scenes/components/VizPanel/VizPanelRenderer.tsx

```diff
@@ -53,10 +55,10 @@ export function VizPanelRenderer({ model }: SceneComponentProps<VizPanel>) {
     $data.setContainerWidth(width);
   }
 
-  if (data?.request?.timeInfo) {
+  if (timeRange instanceof PanelTimeRange) {
     titleItems.push({
       icon: toIconName('clock-nine')!,
-      tooltip: data?.request?.timeInfo,
+      tooltip: timeRange.getTimeOverrideInfo(),
     });
```

Comment:
this should over time be replaced by a full time picker (so when you click it it opens a modal to specify time range for panel)